maven/bnd-indexer-maven-plugin/pom.xml

```diff
@@ -37,7 +37,12 @@
 		</dependency>
 		<dependency>
 			<groupId>biz.aQute.bnd</groupId>
-			<artifactId>org.osgi.impl.bundle.repoindex.cli</artifactId>
+			<artifactId>biz.aQute.bndlib</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.osgi</groupId>
+			<artifactId>org.osgi.core</artifactId>
```

Comment:
bndlib uses osgi.core;version=6, so I think we should use the same here, no?

Also use `<artifactId>osgi.core</artifactId>` which is the proper OSGi artifactId.

Finally, the version number should be set in the parent pom's `<dependencyManagement>` section.